src/tasks/FilterFeedTask.ts

```diff
@@ -78,7 +78,9 @@ export default class FilterFeedTask extends Task {
 				}
 
 				if ( this.publish ) {
-					await NewsUtil.publishMessage( filterFeedMessage );
+					NewsUtil.publishMessage( filterFeedMessage ).catch( err => {
```

Comment:
In theory the catch is not necessary since `NewsUtil.publishMessage` never throws

Reply:
But I guess it's ESLint that then complains?

Reply:
Ah right. I see. I guess that's fine then.

Reply:
Yeah I originally just removed the await until ESLint wanted me to do that as an alternative.